website/docs/sql_ddl.md

```diff
@@ -283,13 +322,7 @@ DROP INDEX [IF EXISTS] index_name ON [TABLE] table_name
 - Both index and column on which the index is created can be qualified with some options in the form of key-value pairs.
   We will see this with an example of functional index below. 
 
-:::note
-Except for the `files`, `column_stats`, `bloom_filters` and `record_index`, all other indexes are experimental. We
-encourage users to try out these features on new tables and provide feedback. Below, we have also listed current
-limitations of these indexes.
-:::
-
-#### Create Functional Index (Experimental)
+#### Create Functional Index
```

Comment:
let's make it Expression Index and add the sql and functions supported here.

Reply:
Addressed